Assets/SteamVR/Input/Editor/SteamVR_Input_PostProcessBuild.cs

```diff
@@ -14,7 +14,9 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
             SteamVR_Input.InitializeFile();
             
             FileInfo fileInfo = new FileInfo(pathToBuiltProject);
-            string buildPath = fileInfo.Directory.FullName;
+            string buildPath = Path.Combine(

```

Comment:
I confirmed that this is a no-op if actionsFilePath has no directory components.